ush/config.sh.RRFS_NA_13km

```diff
@@ -24,20 +24,22 @@ PREDEF_GRID_NAME=GSD_RAP13km
 
 QUILTING="TRUE"
 CCPP_PHYS_SUITE="FV3_GSD_v0"
-FCST_LEN_HRS="12"
+FCST_LEN_HRS="48"
 LBC_SPEC_INTVL_HRS="6"
 
 HALO_BLEND=10
 
-WTIME_RUN_FCST="02:00:00"
+WTIME_RUN_FCST="04:00:00"
```

Comment:
Just checking...is this enough if you increase the forecast length by 4x?

Reply:
It should be.  Recent real-time runs have only required ~50 min to complete a 12-h forecast, so the proposed 4 h wall time is a good scaling from this baseline—and provides a little extra.  Good question, @christinaholtNOAA —I should have provided this detail.